web/docs/operators/sources/context.md

```diff
@@ -0,0 +1,58 @@
+# context
+
+Creates or removes a context. The contexts defined via this source can be used
+with the [`context` transformation](../transformations/context.md).
+
+## Synopsis
+
+```
+context [action] <name>
+
+actions:
+  apply
+  create
+  remove
+  update
+```
+
+## Description
+
+The `context` operator manages and applies contexts. Contexts are a flexible
+mechanism for data enrichment. A context has a type and is implemented as a
+`context plugin`. Each context plugin provides a specific feature set. For
+example the `constants` context can be used to extend events with fields that
+are defined by the key-value-pairs passed to the `context create <name> --type
+constants` operator call.
+
+### `create`
+
+Creates a new context and emits an event that represents it's initial state.
+The schema of that event is specific to the context-type.
+
+#### Synopsis
+
+```
+context create <name> [key=value]
+
+options:
+  --type context-type   The context type to instantiate.
+```
+
+### `remove`
+
+Removes a context and emits a list of the remaining contexts.
+
+#### Synopsis
+
+```
+context remove <name>
```

Comment:
Can you rename this to `context delete`? Matthias and I discussed the choice of verbs earlier today and agreed that delete is clearer, as remove may also just mean that it is made inactive but the state is kept around.

Reply:
I renamed it in https://github.com/tenzir/tenzir-plugins/pull/134/commits/de52c32e87ad01975bdb9b5ff8df67cf4722ebbb